subschemas/substrate-chain/src/typeDefs/democracy.ts

```diff
@@ -27,42 +27,34 @@ export default /* GraphQL */ `
     subCalls: [ProposalSubCall]
   }
 
-  type DemocracyProposal {
-    index: String!
-    balance: String
-    formattedBalance: String
-    seconds: [AccountInfo!]!
+  type Proposal {
+    index: String
     meta: String
     method: String
     section: String
     args: [ProposalArg!]
-    hash: String!
-    proposer: AccountInfo!
-  }
-
-  type DemocracyReferendum {
```

Comment:
All seems good but this is the only thing that leaves me thinking. 
We are merging these two types but they are so different to begin with and seems like many of the fields of the new type are going to be empty because they are only used for one of the types. Does that confirm?

Reply:
@jdomingos 
This indeed was a valid concern. Probably I was jumping into a premature optimization which can be done once we start fetching the proposals from the Polkassembly.

I am closing this PR in favour of this https://github.com/litentry/litentry-graph/pull/216

Thanks 👍